.pre-commit-config.yaml

```diff
@@ -37,3 +42,10 @@ repos:
     rev: v0.15
     hooks:
       - id: validate-pyproject
+
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.6
+    hooks:
+    - id: codespell
+      additional_dependencies:
+        - tomli
```

Comment:
Please consider [`typos`](https://github.com/crate-ci/typos) instead.

Reply:
Added typos in #792